src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    public Figure getRandomFigure() {
+        Random randomTypeOfFigure = new Random();
+        switch (randomTypeOfFigure.nextInt(5)) {
+            case (0):
```

Comment:
```suggestion
            case 0:
```
we can remove brackets